async-servlet-nio-application/gradle.properties

```diff
@@ -0,0 +1,5 @@
+# Install paths
+# -------------
+# Update this to point to a liberty server install
+# This is needed to find the dependencies to build the code and to run the tests
+libertyRoot=
```

Comment:
Just seen this, so I'm guessing that having this means my previous comment is no longer valid, although if someone commented out this line then perhaps the build would fail when searching for file("${libertyRoot}").exists()